rebase_v1.13.1_action_plan.md

```diff
@@ -0,0 +1,81 @@
+# CoreDNS v1.13.1 Rebase Action Plan
+
+_Status legend:_ ✅ complete · 🔄 in progress · ⬜️ pending · 🚫 blocked
```

Comment:
Why would we want to carry a status like this from a rebase to a rebase? Rebase is complex enough on its own as brings a lot of commit work. Things like action plans with statuses may be irrelevant right after the rebase PR is merged. However, we will have to keep this file rolling for each subsequent rebase.

Reply:
Initially I was expecting the rebase PR to have the bare minimum - things we need for building CoreDNS, fixes (if any) and OWNER file changes. All the rest could go into the PR description, kinda like what Grant did the for 1.11.3 rebase. Then we had a gmeet call and you and  Miciah explained that it would be nice to have some report. I think I'm fine with a report and carry consolidations but the action plan kinda stuck in my eye due to its checklist nature (commit audit seems to be redundant too as report covers the part for the carry patch squashing). I'm ok if you and Miciah want to keep a report, I'm fine with having all analysis in the PR description too.

Reply:
I have come around to the dump the report files and put it in the description way of thinking.
I'm  planning on doing this once we're "done" reviewing, or I could do now, but it'll trigger tests:
```bash
    git rm audit_upstream_report.md \
           rebase_v1.13.1_report.md \
           rebase_v1.13.1_action_plan.md \
           carry_consolidation/dependabot_policy.md \
           carry_consolidation/make_test_policy.md \
           carry_consolidation/ocp_dnsnameresolver.md \
           carry_consolidation/vendor_workflow.md
    git commit -m "cleanup: drop rebase planning docs"
```
and then updating the PR description to carry all the content from those docs.

Reply:
this is done with latest commit